src/main/java/com/uber/cadence/internal/replay/ReplayDecisionTaskHandler.java

```diff
@@ -152,7 +153,7 @@ private Result processDecision(PollForDecisionTaskResponse decisionTask) throws
       }
```

Comment:
UI won't let me comment on line 150 but should this be "Throwable"?

Reply:
Fixed